application-templates/starter-typescript/.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
+public
```

Comment:
Perhaps we could do the same in the `@commercetools-applications/merchant-center-template-starter`

Reply:
Ah yes, thanks for noticing!